spec/tools/fix_auth/auth_config_model_spec.rb

```diff
@@ -6,19 +6,10 @@
 
 RSpec.describe FixAuth::AuthConfigModel do
   let(:pass)    { "password" }
-  let(:enc_old) { ManageIQ::Password.new.encrypt(pass, "v2", legacy_key) }
-  let(:enc_new) { ManageIQ::Password.encrypt(pass) }
+  let(:enc_old) { ManageIQ::Password.encrypt(pass, legacy_key) }
 
   let(:legacy_key) { ManageIQ::Password::Key.new }
-  let(:options)    { {:legacy_key => legacy_key} }
-
-  before do
-    ManageIQ::Password.keys["alt"] = legacy_key
-  end
-
-  after do
-    ManageIQ::Password.clear_keys
-  end
```

Comment:
this is nice that we are no longer setting the static state here. So we don't have to clean up after ourselves